testsuite/integration/smoke/pom.xml

```diff
@@ -538,6 +538,14 @@
                         </configuration>
                     </execution>
                 </executions>
+                <!-- Need to override the plugin dependency to make jboss-logging lib work -->
+                <dependencies>
+                    <dependency>
+                        <groupId>org.jboss.logging</groupId>
+                        <artifactId>jboss-logging</artifactId>
+                        <version>${version.org.jboss.logging.jboss-logging}</version>
+                    </dependency>
+                </dependencies>
```

Comment:
Same comment about removing this :)

Reply:
Thanks for taking a look 😃 
I'll wait for the core release and then apply the changes 🙈

Reply:
ohh I didn't realize that it all went so fast and that the core is already out 😃 
I've rebased this one on top of  https://github.com/wildfly/wildfly/pull/18156